_episodes/52.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: episode
+title: "52: Commander Bun Bun takes a bite out of Yugabyte"
+date: 2023-10-26
+tags: trino yugabyte cloud database
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "Episode 52: Commander Bun Bun takes a bite out of Yugabyte"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Denis Magda](https://www.linkedin.com/in/dmagda/), Director of Developer
+  Relations at [Yugabyte](https://www.yugabyte.com/)
+* Anyone else?
+
+## Releases 428-430
+
+Unofficial highlights from Cole:
+
+[Trino 428](https://trino.io/docs/current/release/release-428.html)
+
+* Reduced memory usage for `GROUP BY`
+* Simplified configuration for managing writer counts
+* Faster reads for small Parquet files on data lakes
+* Support for [query options](https://docs.pinot.apache.org/users/user-guide-query/query-options)
+  on dynamic tables in Pinot
+
+[Trino 429](https://trino.io/docs/current/release/release-429.html)
+
+* Faster reading of ORC files in Hive
+* More types supported for schema evolution in Hive
+* Security improvements, including logging out of a session with the Web UI
+
+[Trino 430](https://trino.io/docs/current/release/release-430.html)
+
+* Improved performance of `GROUP BY`
+* Support for setting a timezone on the session level
+* Table statistics in MariaDB
+
+## Concept of the episode: JDBC-based connectors
+
+In Trino, we have a lot of connectors that are based on top of JDBC. JDBC could
+stand for "just da best connectors," but it's really Java database connectivity,
+and it's one of the core APIs by which many of the most prominent connectors in
+the Trino ecosystem function. It's so common, in fact, that we have
+[an example JDBC connector in Trino](/docs/current/develop/example-jdbc.html) to
+make it easier to go implement your own JDBC-based connector if you need one.
+For this episode, we'll be exploring the PostgreSQL connector, which is one of
+the JDBC connectors.
+
+## Concept of the episode: Yugabyte
+
+But if the topic of today's episode is Yugabyte, why are we talking about
+PostgreSQL? Well, if you're unfamiliar with Yugabyte, lifting from
+[their docs](https://docs.yugabyte.com/): "YugabyteDB is distributed PostgreSQL
+that delivers on-demand scale, built-in resilience, and a multi-API interface."
+Distributed architecture should be a familiar concept to a community involved
+with a distributed query engine, and if you understand how Trino is able to
+leverage it, you should also understand why it makes sense to pair with
+Yugabyte. We'll be discussing why Yugabyte got started, what it does differently
```

Comment:
We should also mention that YugabyteDB takes it further than Trino is it is geographically aware and multi cluster and such out of the box from what I understand.. vs just one cluster like Trino

Reply:
I don't think we got into this much in the episode, so I'm going to leave this out of the show notes